ocean_lib/assets/utils.py

```diff
@@ -70,7 +72,9 @@ def add_publisher_trusted_algorithm(
     trusted_algos.append(generate_trusted_algo_dict(ddo=algo_ddo))
     # update with the new list
     privacy_values["publisherTrustedAlgorithms"] = trusted_algos
-    assert compute_service.attributes["main"]["privacy"] == privacy_values
+    assert compute_service.attributes["main"]["privacy"] == privacy_values, (
+        "New trusted algorithm was not added. " "Failed when updating the privacy key. "
```

Comment:
why are you concating two strings on the same line?

Reply:
PyCharm reformatted weird.